src/rules/seczetta-grab-risk-score.js

```diff
@@ -0,0 +1,223 @@
+  // >>             "engagement_type_of_employment": "Corporate Temp",
+  // >>             "personal_state": "AL",
+  // >>             "personal_mobile": "",
+  // >>             "personal_city": "",
+  // >>             "personal_email": "testuser01@seczetta.com",
+  // >>             "personal_zip": "",
+  // >>             "personal_first_name": "Test",
+  // >>             "personal_last_name": "User01",
+  // >>             "personal_street": ""
+  // >>         }
+  // >>     }
+  // >> ] 
+  //  Get Risk Score API 
+  //  >> notice in the response that the risk score still comes back as an array of 1
+  //  >> Request:
+  //  >> https://taylordemo.mynonemployee.com/api/risk_scores?object_id=${objectId}
+  //  >> 
+  //  >> Example Response: 
+  //  >> {
+  //  >>    "risk_scores": [
+  //  >>        {
+  //  >>            "id": "14118693-983e-462f-a330-f3b34d29f281",
+  //  >>            "uid": "036e7e2a3d0c41938609cdc6029d5b11",
+  //  >>            "object_id": "633b5e71-090c-4a47-a1a3-d0b8338df872",
+  //  >>            "object_type": "Profile",
+  //  >>            "overall_score": 3.5,
+  //  >>            "overall_risk_level_id": "29fc4382-2c60-4c6d-891b-15102cdc9e01",
+  //  >>            "impact_score": 7.0,
+  //  >>            "impact_risk_level_id": "29fc4382-2c60-4c6d-891b-15102cdc9e01",
+  //  >>            "probability_score": 0.0,
+  //  >>            "probability_risk_level_id": "c1f10e76-44c0-4bda-b344-8ae2a256d6c4"
+  //  >>        }
+  //  >>    ]
+  //  >> }
+
+  let profileResponse;
+  let riskScoreResponse;
+
+  let attributeId = configuration.SECZETTA_ATTRIBUTE_ID;
+  let profileTypeId = configuration.SECZETTA_PROFILE_TYPE_ID;
+
+  let uid = user.user_name || user.email; //depends on the configuration
+  let profileRequestUrl = configuration.SECZETTA_BASE_URL + '/advanced_search/run';
+  let advancedSearchBody = {
+    advanced_search: {
+      label: "All Contractors",
+      condition_rules_attributes: [
+        {
+          "type": "ProfileTypeRule",
+          "comparison_operator": "==",
+          "value": profileTypeId
+        },
+        {
+          "type": "ProfileAttributeRule",
+          "condition_object_id": attributeId,
+          "object_type": "NeAttribute",
+          "comparison_operator": "==",
+          "value": uid
+        }
+      ]
+    }
+  };
+  
+  try {
+    	profileResponse = await axios.post(profileRequestUrl,advancedSearchBody,{
+        headers: {
+          'Content-Type':'application/json',
+          'Authorization': 'Token token='+configuration.SECZETTA_API_KEY,
+          'Accept': 'application/json'
+        },
+      });
+
+      //if the user isnt found via the advanced search. A 
+    	if( profileResponse.data.profiles.length === 0 ) {
+        console.log("Profile not found. Empty Array sent back!");
+  	 		if( configuration.SECZETTA_AUTHENTICATE_ON_ERROR && configuration.SECZETTA_AUTHENTICATE_ON_ERROR === "true" ) {
```

Comment:
Maybe do something like this at the top of the Rule to make this more clear/DRY?

```
const allowAuthOnError = configuration.SECZETTA_AUTHENTICATE_ON_ERROR === "true"
```